data/presets/emergency/defibrillator.json

```diff
@@ -2,13 +2,14 @@
     "icon": "maki-defibrillator",
     "fields": [
         "indoor",
-        "ref",
```

Comment:
Ref should be kept, it's used e.g. here:
https://osm.mathmos.net/defib/no-coords.html

Reply:
It still can be used, it's just not on the frond-page.
But either way, i can work this in again. Should i?

Reply:
Personally I would have said so as it's used by at least one data consumer. Both post boxes and bus stops (probably all transport) seem to have some form of ref in their field list.

Reply:
Hi, i added the ref again to the editor and made some code corrections in the json format. this commit is added to the pr. https://github.com/chnuessli/id-tagging-schema/commit/b038c43c3a8976d364ccea1005be77dd8e1bc3fa

Reply:
Thanks @chnuessli . FWIW, for me at least this PR seems to have lots of other random changed files too, I don't know if you just need to update or something.